getting-started/git-boot-camp.rst

```diff
@@ -44,11 +44,14 @@ You will only need to do this once.
 
 1. Go to https://github.com/python/cpython.
 
-2. Press ``Fork`` on the top right.
+2. Press ``Fork`` located near the top right of the page.
 
-3. When asked where to fork the repository, choose to fork it to your username.
+   * The required fields ``Owner`` and ``Repository name`` should already be pre-populated with your username and the original repository name "cpython".
```

Comment:
Maybe we can skip this, it's telling about the defaults.
```suggestion
```

Reply:
I'm not sure I agree. While the fields are required, this is explicity stating _how_ the user-specified fields _should_ be to work with the rest of the devguide. 
We don't know, per user, what state will be in (perhaps due to multiple GitHub accounts or previous forks).

Reply:
By default it seems to use the logged (personal?) account, which is what people generally want.

I tried forking a repo that I already forked and I got:
![image](https://github.com/user-attachments/assets/6c319d22-0a94-4733-9cb6-717d07fca59f)

I'm not convinced this sentence is needed -- the repo name is always the same (as stated by the fork page), and if the user has multiple accounts and/or belongs to multiple orgs, GitHub will still use the right default or ask the user to pick one.
Keep in mind that we aim to keep the instruction brief and simple whenever possible.

Reply:
Thanks for checking this and verifying that the fields are unlikely to change. I guess it can be dropped without losing clarity.

I'm not attached to this wording as I am to the unchecking the "Copy the main branch only" button. 

I was approaching this with the mindset that if a field/button _could_ be changed by the user/app, it is good to be clear on what is expected in the fields to minimize possible confusion for a new, less experienced user.

I appreciate the feedback!